website-migration.rst

```diff
@@ -0,0 +1,61 @@
+.. post:: Mar 3, 2023
+   :tags: website, migration 
+   :category: Changelog
+
+.. Categories: https://docs.google.com/document/d/1io3HN61wpFtb6XP-cyONhV5CNWiZ5dbhZM4nvifyT5g/edit#heading=h.3xtrxdnntzqi
+
+Read the Docs website migration to about.readthedocs.com
+========================================================
+
+Read the Docs is in the process of migrating our primary marketing website to a single site: https://about.readthedocs.com.
+This single site allow us to better educate users about which of our products is best for them,
+and direct people to the site where they have registered an account.
+There has been a good deal of confusion about this in the past,
+and this change will make it much more clear about which site you're going to.
+
+Importantly,
+**we are keeping both our Community and Business sites separate for logged in users.**
+Nothing about our commitment to offering free hosting for open source is changing,
+only our approach to explaining our features and products.
+
+Marketing website migration process
+-----------------------------------
+
+The process that we're taking to migrate our marketing site is as follows:
+
+#. ✅ The initial work here was creating a proper marketing site,
```

Comment:
I may remove the emojis from this numbered list and move the 3rd step out of the list and leave it just as a regular paragraph

Reply:
This is meant to be a timeline, and show which steps are done?